util/src/main/java/org/hyperledger/besu/util/Subscribers.java

```diff
@@ -101,17 +102,19 @@ public boolean unsubscribe(final long subscriberId) {
    * @param action the action to perform for each subscriber
    */
   public void forEach(final Consumer<T> action) {
-    subscribers
-        .values()
+    ImmutableSet.copyOf(
+            // we copy here to ensure that our callback, which may add another subscriber on this
```

Comment:
Actually, this is the main change.  Can we move all style changes to a separate PR since they masked where the real change was?

Reply:
removed all the other changes